packages/grafana-ui/src/components/Collapse/CollapsableSection.story.tsx

```diff
@@ -11,13 +13,32 @@ const meta: ComponentMeta<typeof CollapsableSection> = {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['className', 'contentClassName', 'onToggle', 'labelId'],
+    },
+  },
+  args: {
+    isOpen: false,
```

Comment:
In reply to your comment, I think we can just exclude `isOpen` since it doesn't work as a control. Safer than to make changes to how the component works 😅

Reply:
Makes sense, I have added `isOpen` to excluded controls.